app/services/hawkular_proxy_service.rb

```diff
@@ -45,12 +61,27 @@ def get_data(id)
     starts = @params['starts'] || (ends - 8 * 60 * 60 * 1000)
     bucket_duration = @params['bucket_duration'] || nil
     order = @params['order'] || 'ASC'
+    limit = @params['limit'] || 100
 
-    client.gauges.get_data(id,
-                           :limit          => @params['limit'] || 100,
+    data = client.get_data(id,
+                           :limit          => limit.to_i,
                            :starts         => starts.to_i,
                            :ends           => ends.to_i,
                            :bucketDuration => bucket_duration,
                            :order          => order)
+
+    data[0..limit.to_i]
+  end
+
+  def tenants
+    tenants = @cli.hawkular_client.http_get('/tenants')
+    tenants = if @params['include'].blank?
```

Comment:
@yaacov what do you use include for? To limit the list of tenant ids to tenants that you already know about?